src/manifold/src/properties.cpp

```diff
@@ -63,30 +61,27 @@ struct PosMax
 };
 
 struct FiniteVert {
-  bool operator()(glm::vec3 v) { return glm::all(glm::isfinite(v)); }
+  bool operator()(glm::vec3 v) const { return glm::all(glm::isfinite(v)); }
```

Comment:
We can probably just remove a bunch of these tiny functors. I've been turning them into lambdas as I come to them.

Reply:
Agreed. Will fix.